solutions/usaco-966.mdx

```diff
@@ -47,4 +47,46 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<JavaSection>
+  
+```java
+
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+
+class Usaco
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		int N = Integer.parseInt(st.nextToken());
+		
+		Vector<Integer> v = new Vector<Integer>(N);
```

Comment:
Why not `ArrayList`?

Reply:
It can done using both ArrayList and Vector.